src/curated-corpus/components/DuplicateProspectModal/DuplicateProspectModal.styles.tsx

```diff
@@ -1,25 +0,0 @@
-import { createStyles, makeStyles, Theme } from '@material-ui/core/styles';
-
-/**
- * Styles for the DuplicateProspectModal component.
- */
-export const useStyles = makeStyles((theme: Theme) =>
-  createStyles({
-    root: {
-      width: '800px',
-    },
-    link: {
-      color: theme.palette.primary.main,
-      textDecoration: 'none',
-    },
```

Comment:
This style wasn't needed in the duplicate item modal - I think an earlier version of it had a link to the corpus item page, but it has since been replaced with a button. The rest of the styles have been moved over to the component itself into `sx` props.